token-lending/program/tests/flash_borrow_repay.rs

```
@@ -1213,99 +1213,99 @@ async fn test_fail_cpi_repay() {
     );
 }
 
-#[tokio::test]
```

this test fails as expected but the validator panics instead of returning a Err